backend/core/test/listings/listings.e2e-spec.ts

```diff
@@ -52,19 +51,20 @@ describe("Listings", () => {
     // Make the limit 1 less than the full number of listings, so that the first page contains all
     // but the last listing.
     const page = "1"
-    const limit = (getSeedListingsCount() - 1).toString()
-    const params = "/?page=" + page + "&limit=" + limit
+    // This is the number of listings in ../../src/seed.ts minus 1
```

Comment:
Can you add a TODO with a link to the issue?

Reply:
Done!